lib/ansible/config/base.yml

```diff
@@ -1324,6 +1324,30 @@ ERROR_ON_MISSING_HANDLER:
   ini:
   - {key: error_on_missing_handler, section: defaults}
   type: boolean
+CONNECTION_FACTS_MODULES:
```

Comment:
This is a good stopgap for networking until we have `platform`, but I assume this would be superseded by that. Do we want to expose it to env/ini knowing that we'll need to deprecate it whenever we get around to platform, or just keep it as an internal detail for stuff that's shipped in the box for now?

Reply:
since we cannot supply 'maps' that way .. i chose not to  .. at least until ansible.config.yml 

we could create per transport config .. but that does not need to happen right now, I would wait for user feedback.